dist/index.php

```diff
@@ -146,7 +146,7 @@
                         </div>
                     </div>
 
-                    <?php if (!$payIDValidator->hasValidationOccured()) : ?>
+                    <?php if (!$payIDValidator->hasValidationOccurred()) : ?>
```

Comment:
Fix spelling of `occured => occurred`, here and everywhere else.

Reply:
Spelling can be my achilles heel sometimes. :smile: